requirements.txt

```diff
@@ -1,3 +1,4 @@
+websockets==8.1
```

Comment:
Thanks for your contribution.
It would be nice to add a comment in the file stating why this explicit requirement is needed (hopefully it can be removed when/if hbmqtt is updated)

Reply:
I found an issue https://github.com/beerfactory/hbmqtt/issues/239 that is only 16 days old. I feel like this PR might benefit from a few days of wait just in case someone else fixes this?

Reply:
changed to aMQTT, works. 😃